openedx/core/djangoapps/credentials/tests/test_tasks.py

```diff
@@ -107,8 +112,6 @@ def setUp(self):
             self.cert4 = GeneratedCertificateFactory(
                 user=self.user2, course_id='course-v1:edX+Test+4', status=CertificateStatuses.downloadable
             )
-        print(('self.cert1.modified_date', self.cert1.modified_date))
-
```

Comment:
Removed these print statements that seemed like they were used for local debugging.